x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_table_columns.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBasicTableColumn } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiText } from '@elastic/eui';
+import { scaleUpPercentage } from '../../../../components/infrastructure_node_metrics_tables/shared/hooks';
+import { SnapshotNodeMetric } from '../../../../../common/http_api/snapshot_api';
```

Comment:
nit (we're just importing a type here): 
```suggestion
import type { SnapshotNodeMetric } from '../../../../../common/http_api/snapshot_api';
```